src/cow-react/modules/wallet/web3-react/hooks/useEagerlyConnect.ts

```diff
@@ -1,9 +1,10 @@
 import { Connector } from '@web3-react/types'
-import { gnosisSafeConnection, networkConnection } from 'connection'
-import { getConnection } from '@cow/modules/wallet/api/utils'
+import { networkConnection } from '@cow/modules/wallet/web3-react/connection/ network'
```

Comment:
There is an empty space ` network`, I assume this is a typo?

Reply:
Weird, somehow i manage to pre-pend the name of the file with a space! the import was working like that. amazing!
It's fixed here f693626389a33ae47c0c628444e9be79e8e70c32